content/docs/v2/_dev/deployment/frontend-ui.md

```diff
@@ -205,6 +205,14 @@ Field | Description
 
 This will display trace IDs like: `1a2b3c4d5e6f` (instead of the full 32-character ID or default 7-character).
 
+## Tag Search Behavior
+
+Jaeger UI indexes and allows searching for tags (attributes) whose values are strings. If a tag is set as an array (for example, `["value"]`), it may be indexed as a JSON string (e.g., `'["value"]'`). In such cases, you would need to search for the exact JSON string representation in the UI. This can be confusing, as searching for just `value` will not match an array-valued tag; you must search for the full JSON string (e.g., `'["value"]'`).
```

Comment:
> it may be indexed as a JSON string (e.g., `'["value"]'`

are you just restating what I said of did you validate it? Can you search by a string like that?

Reply:
I think it's a bug. I don't want to make changes to documentation before we fix the bug. https://github.com/jaegertracing/jaeger/issues/7346